File: src/Lesson5/java/Ex6.java
package Lesson5.java;
import java.util.Arrays;

public class Ex6 {

    public static boolean arraysAreEqual(int[] arr1, int[] arr2) {

        if (arr1.length != arr2.length) {
            return false;
        }
        for (int i = 0; i < arr1.length; i++) {

            if (arr1[i] != arr2[i]) {
                return false;
            }
        }
        return true;
    }

    public static void main(String[] args) {

        int[] arr1 = {1, 2, 3};
        int[] arr2 = {1, 2, 3};

        System.out.println("First array: " + Arrays.toString(arr1));
        System.out.println("Second array: " + Arrays.toString(arr2));

        boolean result = arraysAreEqual(arr1, arr2);

        System.out.println("Arrays are equal: " + result);
    }
}

File: src/Lesson5/forkotlin/Ex6.kt
package Lesson5.forkotlin

fun arraysAreEqual(arr1: IntArray, arr2: IntArray): Boolean {

    if (arr1.size != arr2.size) {
        return false
    }

    for (i in arr1.indices) {

        if (arr1[i] != arr2[i]) {
            return false
        }
    }

    return true
}

fun main() {

    val arr1 = intArrayOf(1, 2, 3)
    val arr2 = intArrayOf(1, 2, 3)

    println("First array: ${arr1.joinToString()}")
    println("Second array: ${arr2.joinToString()}")

    val result = arraysAreEqual(arr1, arr2)

    println("Arrays are equal: $result")
}